common/src/WooCommerce.php

```diff
@@ -589,4 +594,23 @@ private function isProductReviewsEnabled(): bool {
     private function isSyncedToday(): bool {
         return strtotime($this->getLastReviewSync()) > strtotime('-24 hours');
     }
+
+    public function addOrderDataJsonThankYouPage() {
+        if (is_wc_endpoint_url('order-received')) {
+            if ($this->plugin->getOption('invite') == 3) {
```

Comment:
Let's improve this a bit. It's hard to know which option is 3, would be nice to use a `const` here.